### ISSUE TITLE
UTF8 + B64 not decoded

### ISSUE THREAD

**Issue by soywod** [soywod/rfc2047-decoder#15]
Reference: https://github.com/soywod/himalaya/issues/380

Here the raw string:

```
=?UTF-8?B?UmU6IFlvdXIgQXBw4oCMbOKAjGUgSeKAjEQgaGFzIGJlZW4gbG9ja2XigIzigIxk4oCM4oCMIG9uIFRodXJzZGF5LCBNYXksIDE5IDIwMjIgW3JlZjpfMjU4NzQ2XQ====?=
```

For some reason, it cannot be decoded :thinking: (should be `Re: Your App‌l‌e I‌D has been locke‌‌d‌‌ on Thursday, May, 19 2022 [ref:_258746]`).

**Comment by soywod** [comment]
In fact it is a regression, because I updated the lib inside Himalaya and I have now subject that are not decoded anymore, like:

```
=?utf-8?B?QUdFTkNFIElNTU9CSUxJRVJFIERFUyBWQUxMRUVTIDogSW52aXRhdGlvbiDDoCBzaWduZXIgw6lsZWN0cm9uaXF1ZW1lbnQ=?=
```

or:

```
=?utf-8?B?QUdFTkNFIElNTU9CSUxJRVJFIERFUyBWQUxMRUVTIDogU2lnbmF0dXJlIMOpbGVjdHJvbmlxdWUgZmluYWxpc8OpZQ==?=
```

**Comment by soywod** [comment]
The problems comes from the max length, all those subjects that cannot be decoded contain more than 76 chars #1. So it looks like in theory it is not valid, but in practice email providers do not care about this rule :disappointed:

**Comment by soywod** [comment]
Should be fixed by #17.
